cumulus/client/consensus/aura/src/collators/slot_based/mod.rs

```diff
@@ -102,26 +103,17 @@ pub struct Params<Block, BI, CIDP, Client, Backend, RClient, CHP, Proposer, CS,
 	pub spawner: Spawner,
 }
 
+/// Parameters for slot based aura with export_pov option
+pub struct ParamsWithExport<Block, BI, CIDP, Client, Backend, RClient, CHP, Proposer, CS, Spawner> {
```

Comment:
IMO we don't need an extra set of parameters here. You can just add it to the normal ones.